tests/queries/0_stateless/02117_show_create_table_system.reference

```diff
@@ -367,6 +367,19 @@ CREATE TABLE system.metrics
 )
 ENGINE = SystemMetrics
 COMMENT 'SYSTEM TABLE is built on the fly.'
+CREATE TABLE system.moves
+(
+    `database` String,
+    `table` String,
+    `elapsed` Float64,
+    `target_disk_name` String,
+    `target_disk_path` String,
+    `partition_id` String,
```

Comment:
Let's check this column name.
I think it more like "part_name". Since there is only one part with name "all_3_3_0" under move, and table has one partition with id "all".

Reply:
nice catch, I totally messed it up